test/subscriptionSpec.js

```diff
@@ -81,6 +81,13 @@ describe('Subscriptions', function () {
 
     this.transferUsdPrepared = _.cloneDeep(require('./data/transferUsdPrepared.json'))
     this.transferEurProposed = _.cloneDeep(require('./data/transferEurProposed.json'))
+
+    // Populate the curve cache.
```

Comment:
not needed, since these are all local routes, right? they should already have curves in the routing table when they are created?